stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -112,7 +120,11 @@ spec:
       - name: varlibdockercontainers
         hostPath:
           path: /var/lib/docker/containers
-{{- if .Values.backend.es.tls_ca }}
+{{- if .Values.backend.es.tls_secret }}
+      - name: es-tls-secret
+        secret:
+          secretName: {{ .Values.backend.es.tls_secret }}
+{{- else if .Values.backend.es.tls_ca }}
       - name: es-tls-secret
         secret:
           secretName: "{{ template "fluent-bit.fullname" . }}-es-tls-secret"
```

Comment:
Could you just put the conditional around the secretName line? Everything else seems to be the same.